<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RocketStorageSelectionDialog</class>
 <widget class="QWidget" name="RocketStorageSelectionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>459</width>
    <height>553</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#RocketStorageSelectionDialog {
}

QFrame#controlsFrame {
	background-color: white;
	border-top: 1px solid rgb(207, 207, 207);
}

QLabel, QPushButton {
	color: rgb(80, 80, 80);
	font: 13px &quot;Arial&quot;;
	font-weight: bold;
}

QLabel#infoLabel {
	font: 13px &quot;Arial&quot;;
}

QLineEdit#lineEditFilter {
	font-family: &quot;Arial&quot;;
	font-size: 13px;
	border: 1px solid rgb(154, 154, 154);
	border-radius: 3px;
	min-height: 22px;
	max-height: 22px;
	padding: 0px;
	margin: 0px;
}

QPushButton#buttonSelect, #buttonCancel {
	background-color: transparent;
	color: rgb(80, 80, 80);
	border: 1px solid grey;
	border: 0px;
	border-radius: 3px;
	font-weight: bold;
	min-height: 25px;
	max-height: 25px;
	min-width: 75px;
	max-width: 75px;
	padding-left: 5px;
	padding-right: 5px;
}

QPushButton#buttonSelect::hover, #buttonCancel::hover {
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(232, 234, 233, 255), stop:1 rgba(225, 226, 227, 255));
	color: rgb(60, 60, 60);
	border: 1px solid grey;
}

QPushButton#buttonSelect::pressed, #buttonCancel::pressed {
	background-color: rgba(200, 200, 200);
	border: 1px solid grey;
}

QLineEdit {
	font: 12px &quot;Arial&quot;;
	font-weight: bold;
	color: rgb(80,80,80);
}

QComboBox {
	border: 0px;
	border-bottom: 1px solid grey;
	font-family: &quot;Arial&quot;;
	font-size: 12px;
	color: rgb(65, 65, 65);
	font-weight: bold;
	padding-left: 0px;
    min-width: 100px;
	min-height: 20px;
	max-height: 20px;
	background-color: transparent;
}

QComboBox:hover {
	border-bottom: 1px solid black;
}

QComboBox:disabled, :disabled:hover {
	background-color: rgba(100,100,100,40);
}

QComboBox:down-arrow {
	image: url(':/images/icon-combobox-down-arrow.png');
	width:12px;
}

QComboBox::drop-down {
	subcontrol-origin: padding;
	subcontrol-position: center right;
	padding-right: 2px;
	width: 16px;
	max-height: 11px;
	border: 0px;
}

QComboBox QAbstractItemView {
	border: 1px solid grey;	
	background-color: rgb(255, 255, 255);
	selection-background-color: rgba(230,230,230,120);
	selection-color: black;
	padding: 0px;
}

QComboBox#comboBoxCategory {
	padding-left: 3px;
}

QListView {
	font-family: &quot;Arial&quot;;
	font-size: 11px;
	font-weight: 100;
	show-decoration-selected: 1;
	border: 0px;
/*	background-color: rgba(255,255,255,255); */
	background-color: none;
 }

QListView::item {
	color: rgb(80,80,80);
	min-height: 30px;
	border: 0px;
}

QListView::item:selected, ::item:selected:hover {
	background-color: rgb(230, 230, 230);
	color: black;
}

QListView::item:hover {
	background-color: rgba(230, 230, 230, 70);
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>459</width>
        <height>515</height>
       </rect>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="controlsFrame">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>3</number>
      </property>
      <property name="leftMargin">
       <number>9</number>
      </property>
      <property name="topMargin">
       <number>6</number>
      </property>
      <property name="bottomMargin">
       <number>6</number>
      </property>
      <item>
       <widget class="QLineEdit" name="lineEditFilter">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>24</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>24</height>
         </size>
        </property>
        <property name="placeholderText">
         <string>Filter...</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="infoLabel">
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonCancel">
        <property name="text">
         <string>Cancel</string>
        </property>
        <property name="icon">
         <iconset resource="RocketPlugin.qrc">
          <normaloff>:/images/icon-cross.png</normaloff>:/images/icon-cross.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonSelect">
        <property name="text">
         <string>Select</string>
        </property>
        <property name="icon">
         <iconset resource="RocketPlugin.qrc">
          <normaloff>:/images/icon-check.png</normaloff>:/images/icon-check.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="RocketPlugin.qrc"/>
 </resources>
 <connections/>
</ui>
